app/controllers/fees_controller.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+class FeesController < ApplicationController
+  before_action :set_fee, only: %i[update destroy]
+
+  # GET /fees
+  def index
+    json_response(Fee.all)
+  end
+
+  # POST /fees
+  def create
+    puts create_params
+    fee = Fee.create!(create_params)
+
+    json_response(fee, :created)
+  end
+
+  # PUT /fees/:id
+  def update
+    @fee.update(update_params)
+    json_response(@fee)
+  end
+
+  # DELETE /fees/:id
+  def destroy
+    @fee.destroy
+
+    head :no_content
+  end
+
+  private
+
+  def create_params
+    params.require(:seller_id)
+    update_params
+  end
+
+  def update_params
+    # The multiplier or other attributes such as that of a fee should be
+    # immutable. If you need to modify a fee, set it to inactive and create
+    # a new one. Modifying a fee's multiplier with associated PaymentIntents
+    # could have the side-effect of "re-writing history," changing the fees
```

Comment:
I thought about that. I can serialize the fees, etc. as a JSON and store it on the PaymentIntent when it's created. It is definitely worth pointing out that there is a difference from a "HistoricalFee" and a "CurrentFee". With this model, I am conflating the two. 

The advantage of having it as a separate object as opposed to a serialized object is that we can find all PaymentIntents that were charged with a particular version of a fee. If we serialized it and stored it on the table, it would have to do an N search to find all PaymentIntents that used a particular version of a fee, since there would be no index on that column.

Reply:
I kept it as is, but open to discussing this approach!